tests/integration_tests/performance/test_snapshot_perf.py

```diff
@@ -436,23 +436,39 @@ def _test_snapshot_resume_latency(context):
     file_dumper.dump(result)
 
 
-def _test_older_snapshot_resume_latency(context):
-    builder = context.custom["builder"]
-    logger = context.custom["logger"]
-    snapshot_type = context.custom["snapshot_type"]
-    file_dumper = context.custom["results_file_dumper"]
-    io_engine = context.custom["io_engine"]
+ARTIFACTS = ArtifactCollection(_test_images_s3_bucket())
 
-    firecracker = context.firecracker
-    jailer = firecracker.jailer()
-    jailer.download()
-    fc_version = firecracker.base_name()[1:]
+
+@pytest.mark.parametrize("io_engine", ENGINES)
+@pytest.mark.parametrize(
+    "microvm", ARTIFACTS.microvms(keyword="2vcpu_512mb"), ids=lambda uvm: uvm.name()
+)
+def test_older_snapshot_resume_latency(
+    bin_cloner_path, results_file_dumper, firecracker_release, io_engine, microvm
```

Comment:
oh, ok, I got confused at the beginning because at the original test we were gathering two sets of artifacts, one for the microvm configs and one for the firecracker versions, however here it appeared only the former.

I had to dig a bit to figure out that `firecracker_release` is a fixture.

Could you add this info in the commit body? (In general, I think it's good practice to put a small description in the commit body, even though the change is very simple)

Reply:
Yep, I agree with always having some commit body. Sometimes I am too lazy :D

Reply:
Done